src/components/ExternalLink/index.tsx

```diff
@@ -0,0 +1,5 @@
+import { Link, PropsOf } from "@chakra-ui/react";
+
+export function ExternalLink(props: PropsOf<typeof Link>) {
```

Comment:
While this ExternalLink does share the same props as the base Link, it still needs to be composed with `forwardRef` from `@chakra-ui/react` so that we can pass it to the `as` prop i.e `<Button as={ExternalLink} />`

[Here's](https://github.com/cdklabs/construct-hub-webapp/blob/0d9aaee3bfaa086941dacca7beacd72bda846336/src/components/ExternalLink/ExternalLink.tsx) an implementation you can refer to (needed an external link for one of my feature branches as well)

Additionally, we should spread props after the default prop settings so that we invert control to the developer rather than the component